addons/mail/wizard/mail_compose_message.py

```diff
@@ -65,6 +65,7 @@ def default_get(self, fields):
         result['model'] = result.get('model', self._context.get('active_model'))
         result['res_id'] = result.get('res_id', self._context.get('active_id'))
         result['parent_id'] = result.get('parent_id', self._context.get('message_id'))
+        result['template_id'] = self.env['ir.values'].sudo().get_default('mail.compose.message', 'template_id', condition='mail_compose_act_btn=' + (self.env.context.get('mail_compose_act_btn') or '') + '&' + 'model=' + (result['model'] or '')) or result.get('template_id')
```

Comment:
`ir.values` are now called `ir.default`

Reply:
for code readability, maybe extract the computing of `condition=` to an helper function